config.yaml

```diff
@@ -12,14 +12,8 @@ snmp:
     # mibs_path: "/Users/yling/snmp-project/splunk-connect-for-snmp-mib-server/mibs"
 mongo:
   oid:
-    # host: "localhost"
-    host: "mongo"
-    port: 27017
     database: "snmp_traps"
```

Comment:
I also read the db's name and collection's name from yaml. Can we make them as the Env variable, too?

Reply:
I don't think so as this is not something that Kubernetes manages.

Reply:
Line 7 is also needed to be referred to mib-server URL. e.g localhost needs to be replaced with the host name of mib-server.